storage/storage_tests/helpers.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "geometry/mercator.hpp"
+
+#include "std/unique_ptr.hpp"
+
+namespace storage
+{
+class CountryInfoGetter;
+
+unique_ptr<CountryInfoGetter> CreateCountryInfoGetter();
+unique_ptr<CountryInfoGetter> CreateCountryInfoGetterMigrate();
```

Comment:
Ping on my comment here. How about to merge these two functions into a single function?